when use `go test` , we get:

```c
--- FAIL: Test_addNumbers (0.00s)
    adder_test.go:8: incorrect result: expceted 5, got  4
FAIL
exit status 1
FAIL	adder	0.003s
```

when we fix this problem :

```c
PASS
ok  	adder	0.003s
```
